app/components/windows/EditStreamInfo.vue.ts

```diff
@@ -15,16 +15,26 @@ import { StreamingService } from '../../services/streaming';
 import { WindowsService } from '../../services/windows';
 import { NavigationService } from '../../services/navigation';
 import { CustomizationService } from '../../services/customization';
+import { Multiselect } from 'vue-multiselect';
+import { VideoEncodingOptimizationService } from '../../services/video-encoding-optimizations';
```

Comment:
We've started using absolute paths for imports, since we don't have to change the imports if this file moves.  You can simply do `import { VideoEncodingOptimizationService } from 'services/video-encoding-optimizations';`

Reply:
Done.